src/find/mod.rs

```diff
@@ -219,6 +219,18 @@ mod tests {
 
     use super::*;
 
+    #[cfg(windows)]
```

Comment:
I just wonder whether there are existing libraries for doing this instead of manually implementing it.